### ISSUE TITLE
[Bug]: Waste of time due to incorrect script command

### ISSUE THREAD

**Issue by atljoseph** [lllyasviel/stable-diffusion-webui-forge#835]
### Checklist

- [ ] The issue exists after disabling all extensions
- [X] The issue exists on a clean installation of webui
- [ ] The issue is caused by an extension, but I believe it is caused by a bug in the webui
- [X] The issue exists in the current version of the webui
- [ ] The issue has not been reported before recently
- [ ] The issue has been reported before but has not been fixed yet

### What happened?

Script used to run project clones the EXACT WRONG repo. It clones the exact repo this repo is trying to reform. Please correct it so people don't waste their time and energy, and so that your efforts are more potentiated.

cd "${install_dir}"/ || { printf "\e[1m\e[31mERROR: Can't cd to %s/, aborting...\e[0m" "${install_dir}"; exit 1; }
if [[ -d "${clone_dir}" ]]
then
    cd "${clone_dir}"/ || { printf "\e[1m\e[31mERROR: Can't cd to %s/%s/, aborting...\e[0m" "${install_dir}" "${clone_dir}"; exit 1; }
else
    printf "\n%s\n" "${delimiter}"
    printf "Clone stable-diffusion-webui"
    printf "\n%s\n" "${delimiter}"
    "${GIT}" clone https://github.com/AUTOMATIC1111/stable-diffusion-webui.git "${clone_dir}"
    cd "${clone_dir}"/ || { printf "\e[1m\e[31mERROR: Can't cd to %s/%s/, aborting...\e[0m" "${install_dir}" "${clone_dir}"; exit 1; }
fi

### Steps to reproduce the problem

Use the correct repo url.... The one from THIS repo.

### What should have happened?

Use it and it works right the first time, and not chasing tail.

### Sysinfo

Does not matter. Affects all systems.

### Console logs

```Shell
Doesn't matter. Its clearly wrong.
```

**Comment by DHG-Dav** [comment]
it's not a bug, it's explained here in the readme/presentation... This repo is kind of wrapper around the original webui, giving the exact same results, just changing the way things are handled, to be faster, so it's using the original and keep everything updated from the original...

**Comment by atljoseph** [comment]
Yeah it’s confusing. If I have to remember to correct that line, am not gonna use it. If this is a wrapper around the original repo, then the got command should clone the wrapper and not the thing being wrapped. It was literally starting up with the other repo. After changing the command, all the work you put into the wrapper repo was accessible. Cloning the correct repo is crucial to someone utilizing the code from a given repo. When I download repo A wrapping repo B, and it implicitly downloads and runs code from repo B without running any code from repo A… that’s what I call confusing.
